content/composer/available-actions.md

```diff
@@ -6,42 +6,42 @@ draft: false
 weight: 300
 
 ---
+## Actions Bar
```

Comment:
Since the actions refer to the blueprint, but they are named "Save", "Download" and "Validate", I think it's better to rename this section "Blueprints Action Bar", and rename the sub-titles back ("Save, "Download" and "Validate")